keras/callbacks.py

```diff
@@ -1278,6 +1282,7 @@ def __init__(self,
     self.epochs_since_last_save = 0
     self._batches_seen_since_last_saving = 0
     self._last_batch_seen = 0
+    self.best= initial_value_threshold
```

Comment:
rename best -> initial_value_threshold everywhere, and add a space before the = sign

Reply:
Oh whoops, I see this was named `best` before, and is updated elsewhere. Name is fine! But please do fix the formatting mistake.

Reply:
@mattdangerw 
Thank you for pointing out the mistake. Can you take a look now?